examples/BME280/BME280.ino

```diff
@@ -79,49 +66,41 @@ void readBME280Data() {
 }
 
 void setup() {
-  // Initialize serial:
-  // Serial.begin(9600);
-
-  // check for the presence of the shield:
-  // Serial.print("Configuring WiFi shield/module...\n");
-  if (WiFi.status() == WL_NO_SHIELD) {
-    // Serial.println("WiFi shield not present");
-    // don't continue:
-    while (true)
-      ;
+  Serial.println("Initialize...");
+
+  pinMode(ledPin, OUTPUT);
+  digitalWrite(ledPin, HIGH);
+  Serial.begin(115200);
+  Serial.println("");
+  Serial.print("Connecting to \"");
+  Serial.print(ssid);
+  Serial.println("\"");
+  WiFi.begin(ssid, password);
+  
+  bool blink = true;
+  while (WiFi.status() != WL_CONNECTED) {
+    delay(500);
+    Serial.print(".");
+    digitalWrite(ledPin, blink ? LOW : HIGH); // active low led
+    blink = !blink;
   }
+  digitalWrite(ledPin, HIGH); // active low led
 
-  // configure the LED pin for output mode
-  pinMode(LED_BUILTIN, OUTPUT);
+  Serial.println("");
+  Serial.print("Connected to ");
+  Serial.println(ssid);
+  Serial.print("IP address: ");
+  Serial.println(WiFi.localIP());
 
   Wire.begin();
   while (!bme.begin()) {
-    // Serial.println("Could not find BME280I2C sensor!");
+    Serial.println("Could not find BME280I2C sensor!");
     delay(1000);
   }
-
-  // Serial.println("Starting in provisioning mode...");
-  // Start in provisioning mode:
-  //  1) This will try to connect to a previously associated access point.
-  //  2) If this fails, an access point named "wifi101-XXXX" will be created,
-  //  where XXXX
-  //     is the last 4 digits of the boards MAC address. Once you are connected
-  //     to the access point, you can configure an SSID and password by
-  //     visiting http://wifi101/
-  WiFi.beginProvision();
-
-  while (WiFi.status() != WL_CONNECTED) {
-    // wait while not connected
-
-    // blink the led to show an unconnected status
-    digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
-    delay(500);
-    digitalWrite(LED_BUILTIN, PIN_STATE_LOW);
-    delay(500);
-  }
+  Serial.println("BME280I2C connected and initialized.");
 
   // connected, make the LED stay on
-  digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
+  //digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
```

Comment:
Why don't we keep this visual feedback?

Reply:
Fair enough. Don't remember why I commented it out ^^

Reply:
Fixed it.